iplimit.py

```diff
@@ -0,0 +1,121 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# by Dan Parsons, aka lerxst, <dparsons@mozilla.com>
+
+from flask import Flask
+from flask import request
+from flaskext.mysql import MySQL
+from netaddr import IPNetwork, IPAddress, core
+import time
+import datetime
+import json
+import ConfigParser
+import optparse
+import sys
+import os
+
+# in days
+DEFAULT_EXCEPTION_LENGTH=2
```

Comment:
you might want to make it configurable so that you dont need to redeploy if it has to be changed, since theres a config file anyway